packages/terra-table/src/utils/GridContext.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const GridContext = React.createContext({
+  role: 'table',
+  cellAriaLiveMessage: '',
```

Comment:
This property should not be required.  Nobody is reading the value.  The table just needs the ability to call the callback.

Reply:
Done.